helm-chart/values.yaml

```diff
@@ -364,56 +355,30 @@ grafana:
       enabled: false
 
 ingress:
+  ## @param ingress.enabled Enable ingress record generation for CVAT
+  ##
   enabled: false
-
-# In case you need an ingress, write them manually in your my.values.eml, see example below:
-#  enabled: true
-#  annotations:
-#    traefik.ingress.kubernetes.io/router.entrypoints: web
-#    kubernetes.io/ingress.class: traefik
-#  hosts:
-#    - host: cvat.local
-#      paths:
-#        - path: /api
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /admin
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /static
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /django-rq
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /profiler
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path : /
-#          pathType: "Prefix"
-#          service:
-#            name: frontend-service
-#            port: 80
-#  tls:
-#    - hosts:
-#      - <your_domain>
-#    secretName: ingress-tls-cvat
```

Comment:
What about these parameters? They should either be documented, or their usages should be removed.

Reply:
removed from ingress

Reply:
Then how would the user configure TLS in a Traefik-less configuration? I think you should provide some replacement for these parameres.

Reply:
TLS has never worked before, I guess it's a separate task to add tls support out of the box

Reply:
Ok, you are right, let's keep the part described in the spec, but anyway the tls configuration is too controller specific